modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/paths/APIBuilder.path

```diff
@@ -105,6 +110,16 @@
 	<td>//input[(@placeholder='Enter title.')]</td>
 	<td></td>
 </tr>
+<tr>
```

Comment:
I've tried locally that seems the MODAL_DELETE_BUTTON and DELETE_BUTTON_DISABLED are the same Delete button in modal, could you use one?

Reply:
I don't think so, In the testcase I need to test the button status is disabled

Reply:
I found we can use button#ANY_DISABLED to replace the DELETE_BUTTON_DISABLED